frontend/src/style/bootstrap/attivio-spinner.less

```diff
@@ -0,0 +1,12 @@
+
+.attivio-spinner {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
It seems from this page that transform: translate() is supported in all of the browsers w support... if so, the various vendor-specific values shouldn't be necessary...
https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function